Dockerfile

```diff
@@ -1,18 +1,46 @@
-ARG REQUIREMENTS_MD5
-ARG DOCKER_REGISTRY_IDENTIFER
-FROM ${DOCKER_REGISTRY_IDENTIFER}/hyperion-base:${REQUIREMENTS_MD5}
+FROM ghcr.io/astral-sh/uv:python3.12-trixie-slim
 
+# Default number of workers; can be overridden at runtime
+ENV WORKERS=1
+
+# Update package list and install weasyprint dependencies
+RUN apt-get update && apt-get install -y \
+    weasyprint \
+    && rm -rf /var/lib/apt/lists/*

```

Comment:
Could you add comments explaining why we install weasyprint manually, and why we need to delete a folder?